homeassistant/components/filesize/sensor.py

```diff
@@ -74,13 +74,12 @@ async def async_setup_entry(
     coordinator = FileSizeCoordinator(hass, fullpath)
     await coordinator.async_config_entry_first_refresh()
 
-    if get_path.exists() and get_path.is_file():
-        async_add_entities(
-            [
-                FilesizeEntity(description, fullpath, entry.entry_id, coordinator)
-                for description in SENSOR_TYPES
-            ]
-        )
+    async_add_entities(
+        [
+            FilesizeEntity(description, fullpath, entry.entry_id, coordinator)
+            for description in SENSOR_TYPES
+        ]
```

Comment:
```suggestion
            FilesizeEntity(description, fullpath, entry.entry_id, coordinator)
            for description in SENSOR_TYPES
```

black will complain about this suggestion.